src/index.html

```diff
@@ -4,14 +4,159 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action = "https://mate-academy-form-lesson.herokuapp.com/create-application " method = "post">
+      <fieldset class = "group">
+        <legend>Personal information</legend>
+        <label class = "form_field">Surname
+          <input
+            type="text"
+            class = "field"
+            name = "surname"
```

Comment:
```suggestion
            class="field"
            name="surname"
```
не потрібно пробілів тут